City-of-Helsinki/helsinki-paatokset
# PP-127: PP-127: created decision tree block

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-127

Created decision tree block that renders on a page that has /paatoksentekoprosessi as path

**How to test**

Checkout branch and run `drush cim -y; drush cr`

Create a landing page called "Päätöksentekoprosessi". The decision tree should appear there as the last element if there is some content added through the node editing. Check the styles etc that they match with the design. 

The arrows are the ones that I've created so they could be changed to actual svg:s made by the designer so that they match the design properly.
